docs/modules/vertiq_23xx_family.rst

```diff
@@ -0,0 +1,56 @@
+
+.. _vertiq_23xx_family:
+
+*********************************
+Vertiq 23-XX Family 
+*********************************
+
+.. csv-table:: Vertiq 23-XX Family of Modules
+        :header: "Size", "Kv", "Default Firmware", "Available Firmware"
+        :align: left
+
+        "23-06", "220", "Servo", "Servo, Step/Direction"
```

Comment:
Can you ditch step/direction? Trying to not make that a thing anymore, we've taken it off the firmwares on the website: https://www.vertiq.co/23-06